### PR TITLE
Try to clarify the sections about cubemaps

### PR DESCRIPTION
**Pull request by pixelflinger**

### CODE REVIEW

File: docs/Filament.md.html [hunk 1]
```diff
@@ -3314,25 +3314,51 @@
...
+### World coordinates system
 
 Filament uses a Y-up, right-handed coordinate system.
 
 ![Figure [coordinates]: Red +X, green +Y, blue +Z (rendered in Marmoset Toolbag).](images/screenshot_coordinates.jpg)
 
+
+### Camera coordinates system
+
+Filament's `Camera` look towards its local -Z axis. That is, when placing a camera in the world
```
Comment (prideout) [inline docs/Filament.md.html]: s/look/looks/

File: docs/Filament.md.html [hunk 2]
```diff
@@ -3314,25 +3314,51 @@
...
+without any transform applied to it, the camera looks down the world's -Z axis.
+
+
 ### Cubemaps coordinates system
 
-All the cubemaps used in Filament (environment background, reflection probes, etc.) will follow the OpenGL convention for faces alignment show in figure [cubemapCoordinates].
+All cubemaps used in Filament through the `Texture` class follow the OpenGL convention for faces alignment shown in figure [cubemapCoordinates].
```
Comment (prideout) [inline docs/Filament.md.html]: s/faces/face/

File: docs/Filament.md.html [hunk 3]
```diff
@@ -3314,25 +3314,51 @@
...
-#### Equirectangular environment maps
+Note that environment background and reflection probes are mirrored (see section [Mirroring]).
 
-To convert equirectangular environment maps to horizontal/vertical cross cubemaps we position the +Z face in the center of the source rectilinear environment map.
 
 #### Mirroring
 
-To simplify the rendering of reflections, cubemaps will be stored mirrored on the X axis. This means that cubemaps used as environment backgrounds need to be mirrored again at runtime. An easy way to achieve for skyboxes is to use textured back faces.
+To simplify the rendering of reflections, IBL cubemaps are stored mirrored on the X axis. This is
+the default behaviour of the `cmgen` tool. This means that IBL cubemaps used as environment
```
Comment (prideout) [inline docs/Filament.md.html]: "This means that an IBL cubemap used as an environment background needs"...

File: docs/Filament.md.html [hunk 4]
```diff
@@ -3314,25 +3314,52 @@
...
+
+#### Equirectangular environment maps
+
+To convert equirectangular environment maps to horizontal/vertical cross cubemaps we position the
++Z face in the center of the source rectilinear environment map.
+
+
+#### World space orientation of environment maps and Skyboxes
+
+When using the `IndirectLight` and `Skybox` classes, the specified cubemap is oriented such that its
```
Comment (romainguy) [inline docs/Filament.md.html]: We shouldn't mention implementation details (like class names) in this document.

**Pull request merged**